client/src/Components/Common/Button.style.js

```diff
@@ -20,13 +20,15 @@ export const Button = styled.button`
   ${sharedStyles};
   a {
     text-decoration: none;
-    color: var(--black);
+    color: ${({ theme }) => theme.colors.black};
   }
 
-  &:hover {
-    background-color: var(--secondary);
-    color: var(--white);
-  }
+  ${props =>
+    !props.disabled &&
+    ` &:hover {
+      background-color: #e8529b;
```

Comment:
```suggestion
      background-color: ${props.theme.colors.pink};
```

you can get it to work like this